llumnix/queue/queue_client_base.py

```diff
@@ -18,5 +18,5 @@
 
 class QueueClientBase(ABC):
     @abstractmethod
-    async def put_nowait_batch(self, items: Iterable, server_info: ServerInfo):
+    async def put_nowait(self, items: Iterable, server_info: ServerInfo):
```

Comment:
consider add a new method named put_nowait

put_nowait(self, items: Iterable ....    ->. put_nowait(self, item: Iterable ....    ， and other places

Reply:
Any maybe better

Reply:
Done.